backend/routes/users.js

```diff
@@ -11,6 +13,9 @@ router.get('/getUser', authJwt.verifyToken, (req, res) => {
   authController.getAuthUser(req, res);
 });
 
+router.get('/role', userController.getAllRoles);
+router.patch('/role', adminController.updateRole);
```

Comment:
The route updateRole (line 17) is already called in the admin route, can we remove it if not necessary.

Reply:
The Admin route doesn't properly work so this route is used for the moment